/work2/09808/mp46753/frontera/ML_Project/VQGraph/vq.py:283: FutureWarning: `torch.cuda.amp.autocast(args...)` is deprecated. Please use `torch.amp.autocast('cuda', args...)` instead.
  @autocast(enabled=False)
/work2/09808/mp46753/frontera/ML_Project/VQGraph/vq.py:418: FutureWarning: `torch.cuda.amp.autocast(args...)` is deprecated. Please use `torch.amp.autocast('cuda', args...)` instead.
  @autocast(enabled=False)
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/dataloading/dataloader.py:1144: DGLWarning: Dataloader CPU affinity opt is not enabled, consider switching it on (see enable_cpu_affinity() or CPU best practices for DGL [https://docs.dgl.ai/tutorials/cpu/cpu_best_practises.html])
  dgl_warning(
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:92: DGLWarning: Recommend creating graphs by `dgl.graph(data)` instead of `dgl.DGLGraph(data)`.
  dgl_warning(
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/torch/_dynamo/variables/functions.py:663: UserWarning: Graph break due to unsupported builtin torch._C._to_dlpack. This function is either a Python builtin (e.g. _warnings.warn) or a third-party C/C++ Python extension (perhaps created with pybind). If it is a Python builtin, please file an issue on GitHub so the PyTorch team can add support for it and see the next case for a workaround. If it is a third-party C/C++ Python extension, please either wrap it into a PyTorch-understood custom operator (see https://pytorch.org/tutorials/advanced/custom_ops_landing_page.html for more details) or, if it is traceable, use torch.compiler.allow_in_graph.
  torch._dynamo.utils.warn_once(msg)
V0421 02:16:50.072470 47728129008896 torch/_dynamo/guards.py:2611] [4/1] [__recompiles] Recompiling function create_unitgraph_from_coo in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph_index.py:1254
V0421 02:16:50.072470 47728129008896 torch/_dynamo/guards.py:2611] [4/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:16:50.072470 47728129008896 torch/_dynamo/guards.py:2611] [4/1] [__recompiles]     - tensor 'L['row']' dispatch key set mismatch. expected DispatchKeySet(CPU, BackendSelect, ADInplaceOrView, AutogradCPU), actual DispatchKeySet(CUDA, BackendSelect, ADInplaceOrView, AutogradCUDA)
V0421 02:16:50.087660 47728129008896 torch/_dynamo/guards.py:2611] [5/1] [__recompiles] Recompiling function to_dgl_nd in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/__init__.py:141
V0421 02:16:50.087660 47728129008896 torch/_dynamo/guards.py:2611] [5/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:16:50.087660 47728129008896 torch/_dynamo/guards.py:2611] [5/1] [__recompiles]     - tensor 'L['data']' dispatch key set mismatch. expected DispatchKeySet(CPU, BackendSelect, ADInplaceOrView, AutogradCPU), actual DispatchKeySet(CUDA, BackendSelect, ADInplaceOrView, AutogradCUDA)
V0421 02:16:50.096446 47728129008896 torch/_dynamo/guards.py:2611] [6/1] [__recompiles] Recompiling function zerocopy_to_dgl_ndarray in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:432
V0421 02:16:50.096446 47728129008896 torch/_dynamo/guards.py:2611] [6/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:16:50.096446 47728129008896 torch/_dynamo/guards.py:2611] [6/1] [__recompiles]     - tensor 'L['data']' dispatch key set mismatch. expected DispatchKeySet(CPU, BackendSelect, ADInplaceOrView, AutogradCPU), actual DispatchKeySet(CUDA, BackendSelect, ADInplaceOrView, AutogradCUDA)
V0421 02:16:50.127631 47728129008896 torch/_dynamo/guards.py:2611] [8/1] [__recompiles] Recompiling function torch_dynamo_resume_in_create_unitgraph_from_coo_at_1296 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph_index.py:1296
V0421 02:16:50.127631 47728129008896 torch/_dynamo/guards.py:2611] [8/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:16:50.127631 47728129008896 torch/_dynamo/guards.py:2611] [8/1] [__recompiles]     - tensor 'L['col']' dispatch key set mismatch. expected DispatchKeySet(CPU, BackendSelect, ADInplaceOrView, AutogradCPU), actual DispatchKeySet(CUDA, BackendSelect, ADInplaceOrView, AutogradCUDA)
W0421 02:16:57.298130 47728129008896 torch/_dynamo/variables/tensor.py:715] [30/0] Graph break from `Tensor.item()`, consider setting:
W0421 02:16:57.298130 47728129008896 torch/_dynamo/variables/tensor.py:715] [30/0]     torch._dynamo.config.capture_scalar_outputs = True
W0421 02:16:57.298130 47728129008896 torch/_dynamo/variables/tensor.py:715] [30/0] or:
W0421 02:16:57.298130 47728129008896 torch/_dynamo/variables/tensor.py:715] [30/0]     env TORCHDYNAMO_CAPTURE_SCALAR_OUTPUTS=1
W0421 02:16:57.298130 47728129008896 torch/_dynamo/variables/tensor.py:715] [30/0] to include these operations in the captured graph.
W0421 02:16:57.298130 47728129008896 torch/_dynamo/variables/tensor.py:715] [30/0] 
V0421 02:17:00.151782 47728129008896 torch/_dynamo/guards.py:2611] [4/2] [__recompiles] Recompiling function create_unitgraph_from_coo in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph_index.py:1254
V0421 02:17:00.151782 47728129008896 torch/_dynamo/guards.py:2611] [4/2] [__recompiles]     triggered by the following guard failure(s):
V0421 02:17:00.151782 47728129008896 torch/_dynamo/guards.py:2611] [4/2] [__recompiles]     - tensor 'L['row']' size mismatch at index 0. expected 0, actual 830
V0421 02:17:00.151782 47728129008896 torch/_dynamo/guards.py:2611] [4/2] [__recompiles]     - tensor 'L['row']' dispatch key set mismatch. expected DispatchKeySet(CPU, BackendSelect, ADInplaceOrView, AutogradCPU), actual DispatchKeySet(CUDA, BackendSelect, ADInplaceOrView, AutogradCUDA)
V0421 02:17:00.179509 47728129008896 torch/_dynamo/guards.py:2611] [5/2] [__recompiles] Recompiling function to_dgl_nd in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/__init__.py:141
V0421 02:17:00.179509 47728129008896 torch/_dynamo/guards.py:2611] [5/2] [__recompiles]     triggered by the following guard failure(s):
V0421 02:17:00.179509 47728129008896 torch/_dynamo/guards.py:2611] [5/2] [__recompiles]     - tensor 'L['data']' size mismatch at index 0. expected 0, actual 830
V0421 02:17:00.179509 47728129008896 torch/_dynamo/guards.py:2611] [5/2] [__recompiles]     - tensor 'L['data']' dispatch key set mismatch. expected DispatchKeySet(CPU, BackendSelect, ADInplaceOrView, AutogradCPU), actual DispatchKeySet(CUDA, BackendSelect, ADInplaceOrView, AutogradCUDA)
V0421 02:17:00.191270 47728129008896 torch/_dynamo/guards.py:2611] [6/2] [__recompiles] Recompiling function zerocopy_to_dgl_ndarray in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:432
V0421 02:17:00.191270 47728129008896 torch/_dynamo/guards.py:2611] [6/2] [__recompiles]     triggered by the following guard failure(s):
V0421 02:17:00.191270 47728129008896 torch/_dynamo/guards.py:2611] [6/2] [__recompiles]     - tensor 'L['data']' size mismatch at index 0. expected 0, actual 830
V0421 02:17:00.191270 47728129008896 torch/_dynamo/guards.py:2611] [6/2] [__recompiles]     - tensor 'L['data']' dispatch key set mismatch. expected DispatchKeySet(CPU, BackendSelect, ADInplaceOrView, AutogradCPU), actual DispatchKeySet(CUDA, BackendSelect, ADInplaceOrView, AutogradCUDA)
V0421 02:17:00.229689 47728129008896 torch/_dynamo/guards.py:2611] [8/2] [__recompiles] Recompiling function torch_dynamo_resume_in_create_unitgraph_from_coo_at_1296 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph_index.py:1296
V0421 02:17:00.229689 47728129008896 torch/_dynamo/guards.py:2611] [8/2] [__recompiles]     triggered by the following guard failure(s):
V0421 02:17:00.229689 47728129008896 torch/_dynamo/guards.py:2611] [8/2] [__recompiles]     - tensor 'L['col']' size mismatch at index 0. expected 0, actual 830
V0421 02:17:00.229689 47728129008896 torch/_dynamo/guards.py:2611] [8/2] [__recompiles]     - tensor 'L['col']' dispatch key set mismatch. expected DispatchKeySet(CPU, BackendSelect, ADInplaceOrView, AutogradCPU), actual DispatchKeySet(CUDA, BackendSelect, ADInplaceOrView, AutogradCUDA)
V0421 02:17:00.246813 47728129008896 torch/_dynamo/guards.py:2611] [32/1] [__recompiles] Recompiling function torch_dynamo_resume_in_add_edges_at_574 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:574
V0421 02:17:00.246813 47728129008896 torch/_dynamo/guards.py:2611] [32/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:17:00.246813 47728129008896 torch/_dynamo/guards.py:2611] [32/1] [__recompiles]     - L['___stack0'] == 683                                       
V0421 02:17:00.460824 47728129008896 torch/_dynamo/guards.py:2611] [34/1] [__recompiles] Recompiling function cat in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:222
V0421 02:17:00.460824 47728129008896 torch/_dynamo/guards.py:2611] [34/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:17:00.460824 47728129008896 torch/_dynamo/guards.py:2611] [34/1] [__recompiles]     - tensor 'L['seq'][0]' size mismatch at index 0. expected 0, actual 830
V0421 02:17:01.199188 47728129008896 torch/_dynamo/guards.py:2611] [20/1] [__recompiles] Recompiling function add_edges in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:435
V0421 02:17:01.199188 47728129008896 torch/_dynamo/guards.py:2611] [20/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:17:01.199188 47728129008896 torch/_dynamo/guards.py:2611] [20/1] [__recompiles]     - tensor 'L['u']' size mismatch at index 0. expected 830, actual 189
V0421 02:17:01.220470 47728129008896 torch/_dynamo/guards.py:2611] [26/1] [__recompiles] Recompiling function torch_dynamo_resume_in_add_edges_at_541 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:541
V0421 02:17:01.220470 47728129008896 torch/_dynamo/guards.py:2611] [26/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:17:01.220470 47728129008896 torch/_dynamo/guards.py:2611] [26/1] [__recompiles]     - tensor 'L['v']' size mismatch at index 0. expected 830, actual 189
V0421 02:17:01.248462 47728129008896 torch/_dynamo/guards.py:2611] [27/1] [__recompiles] Recompiling function torch_dynamo_resume_in_add_edges_at_542 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:542
V0421 02:17:01.248462 47728129008896 torch/_dynamo/guards.py:2611] [27/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:17:01.248462 47728129008896 torch/_dynamo/guards.py:2611] [27/1] [__recompiles]     - tensor 'L['___stack0']' size mismatch at index 0. expected 830, actual 189
V0421 02:17:01.264802 47728129008896 torch/_dynamo/guards.py:2611] [28/1] [__recompiles] Recompiling function torch_dynamo_resume_in_add_edges_at_545 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:545
V0421 02:17:01.264802 47728129008896 torch/_dynamo/guards.py:2611] [28/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:17:01.264802 47728129008896 torch/_dynamo/guards.py:2611] [28/1] [__recompiles]     - tensor 'L['u']' size mismatch at index 0. expected 830, actual 189
V0421 02:17:01.330211 47728129008896 torch/_dynamo/guards.py:2611] [30/1] [__recompiles] Recompiling function torch_dynamo_resume_in_add_edges_at_573 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:573
V0421 02:17:01.330211 47728129008896 torch/_dynamo/guards.py:2611] [30/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:17:01.330211 47728129008896 torch/_dynamo/guards.py:2611] [30/1] [__recompiles]     - tensor 'L['u']' size mismatch at index 0. expected 830, actual 189
V0421 02:17:01.789719 47728129008896 torch/_dynamo/guards.py:2611] [32/2] [__recompiles] Recompiling function torch_dynamo_resume_in_add_edges_at_574 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:574
V0421 02:17:01.789719 47728129008896 torch/_dynamo/guards.py:2611] [32/2] [__recompiles]     triggered by the following guard failure(s):
V0421 02:17:01.789719 47728129008896 torch/_dynamo/guards.py:2611] [32/2] [__recompiles]     - tensor 'L['v']' size mismatch at index 0. expected 830, actual 189
V0421 02:17:01.789719 47728129008896 torch/_dynamo/guards.py:2611] [32/2] [__recompiles]     - L['___stack0'] == 683                                       
V0421 02:17:02.039076 47728129008896 torch/_dynamo/guards.py:2611] [34/2] [__recompiles] Recompiling function cat in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:222
V0421 02:17:02.039076 47728129008896 torch/_dynamo/guards.py:2611] [34/2] [__recompiles]     triggered by the following guard failure(s):
V0421 02:17:02.039076 47728129008896 torch/_dynamo/guards.py:2611] [34/2] [__recompiles]     - tensor 'L['seq'][1]' size mismatch at index 0. expected 830, actual 189
V0421 02:17:02.039076 47728129008896 torch/_dynamo/guards.py:2611] [34/2] [__recompiles]     - tensor 'L['seq'][0]' size mismatch at index 0. expected 0, actual 1660
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/torch/_dynamo/variables/functions.py:663: UserWarning: Graph break due to unsupported builtin posix.stat. This function is either a Python builtin (e.g. _warnings.warn) or a third-party C/C++ Python extension (perhaps created with pybind). If it is a Python builtin, please file an issue on GitHub so the PyTorch team can add support for it and see the next case for a workaround. If it is a third-party C/C++ Python extension, please either wrap it into a PyTorch-understood custom operator (see https://pytorch.org/tutorials/advanced/custom_ops_landing_page.html for more details) or, if it is traceable, use torch.compiler.allow_in_graph.
  torch._dynamo.utils.warn_once(msg)
V0421 02:17:06.251185 47728129008896 torch/_dynamo/guards.py:2611] [6/3] [__recompiles] Recompiling function zerocopy_to_dgl_ndarray in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:432
V0421 02:17:06.251185 47728129008896 torch/_dynamo/guards.py:2611] [6/3] [__recompiles]     triggered by the following guard failure(s):
V0421 02:17:06.251185 47728129008896 torch/_dynamo/guards.py:2611] [6/3] [__recompiles]     - GLOBAL_STATE changed: grad_mode 
V0421 02:17:06.251185 47728129008896 torch/_dynamo/guards.py:2611] [6/3] [__recompiles]     - GLOBAL_STATE changed: grad_mode 
V0421 02:17:06.251185 47728129008896 torch/_dynamo/guards.py:2611] [6/3] [__recompiles]     - GLOBAL_STATE changed: grad_mode 
V0421 02:17:06.297006 47728129008896 torch/_dynamo/guards.py:2611] [7/1] [__recompiles] Recompiling function torch_dynamo_resume_in_zerocopy_to_dgl_ndarray_at_435 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:435
V0421 02:17:06.297006 47728129008896 torch/_dynamo/guards.py:2611] [7/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:17:06.297006 47728129008896 torch/_dynamo/guards.py:2611] [7/1] [__recompiles]     - GLOBAL_STATE changed: grad_mode 
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/torch/_dynamo/variables/functions.py:663: UserWarning: Graph break due to unsupported builtin time.perf_counter. This function is either a Python builtin (e.g. _warnings.warn) or a third-party C/C++ Python extension (perhaps created with pybind). If it is a Python builtin, please file an issue on GitHub so the PyTorch team can add support for it and see the next case for a workaround. If it is a third-party C/C++ Python extension, please either wrap it into a PyTorch-understood custom operator (see https://pytorch.org/tutorials/advanced/custom_ops_landing_page.html for more details) or, if it is traceable, use torch.compiler.allow_in_graph.
  torch._dynamo.utils.warn_once(msg)
V0421 02:17:06.539854 47728129008896 torch/_dynamo/guards.py:2611] [86/1] [__recompiles] Recompiling function torch_dynamo_resume_in_hook_at_12 in /work2/09808/mp46753/frontera/ML_Project/VQGraph/benchmark_utils.py:12
V0421 02:17:06.539854 47728129008896 torch/_dynamo/guards.py:2611] [86/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:17:06.539854 47728129008896 torch/_dynamo/guards.py:2611] [86/1] [__recompiles]     - L['___stack0'] == 923268.01162549                           
V0421 02:17:06.635847 47728129008896 torch/_dynamo/guards.py:2611] [86/2] [__recompiles] Recompiling function torch_dynamo_resume_in_hook_at_12 in /work2/09808/mp46753/frontera/ML_Project/VQGraph/benchmark_utils.py:12
V0421 02:17:06.635847 47728129008896 torch/_dynamo/guards.py:2611] [86/2] [__recompiles]     triggered by the following guard failure(s):
V0421 02:17:06.635847 47728129008896 torch/_dynamo/guards.py:2611] [86/2] [__recompiles]     - L['___stack0'] == 923268.024748108                          
V0421 02:17:06.635847 47728129008896 torch/_dynamo/guards.py:2611] [86/2] [__recompiles]     - L['___stack0'] == 923268.01162549                           
V0421 02:17:09.177930 47728129008896 torch/_dynamo/guards.py:2611] [86/3] [__recompiles] Recompiling function torch_dynamo_resume_in_hook_at_12 in /work2/09808/mp46753/frontera/ML_Project/VQGraph/benchmark_utils.py:12
V0421 02:17:09.177930 47728129008896 torch/_dynamo/guards.py:2611] [86/3] [__recompiles]     triggered by the following guard failure(s):
V0421 02:17:09.177930 47728129008896 torch/_dynamo/guards.py:2611] [86/3] [__recompiles]     - L['___stack0'] == 923268.120714279                          
V0421 02:17:09.177930 47728129008896 torch/_dynamo/guards.py:2611] [86/3] [__recompiles]     - L['___stack0'] == 923268.024748108                          
V0421 02:17:09.177930 47728129008896 torch/_dynamo/guards.py:2611] [86/3] [__recompiles]     - L['___stack0'] == 923268.01162549                           
V0421 02:17:11.004952 47728129008896 torch/_dynamo/guards.py:2611] [86/4] [__recompiles] Recompiling function torch_dynamo_resume_in_hook_at_12 in /work2/09808/mp46753/frontera/ML_Project/VQGraph/benchmark_utils.py:12
V0421 02:17:11.004952 47728129008896 torch/_dynamo/guards.py:2611] [86/4] [__recompiles]     triggered by the following guard failure(s):
V0421 02:17:11.004952 47728129008896 torch/_dynamo/guards.py:2611] [86/4] [__recompiles]     - L['___stack0'] == 923270.662628611                          
V0421 02:17:11.004952 47728129008896 torch/_dynamo/guards.py:2611] [86/4] [__recompiles]     - L['___stack0'] == 923268.120714279                          
V0421 02:17:11.004952 47728129008896 torch/_dynamo/guards.py:2611] [86/4] [__recompiles]     - L['___stack0'] == 923268.024748108                          
V0421 02:17:11.004952 47728129008896 torch/_dynamo/guards.py:2611] [86/4] [__recompiles]     - L['___stack0'] == 923268.01162549                           
V0421 02:17:11.073667 47728129008896 torch/_dynamo/guards.py:2611] [86/5] [__recompiles] Recompiling function torch_dynamo_resume_in_hook_at_12 in /work2/09808/mp46753/frontera/ML_Project/VQGraph/benchmark_utils.py:12
V0421 02:17:11.073667 47728129008896 torch/_dynamo/guards.py:2611] [86/5] [__recompiles]     triggered by the following guard failure(s):
V0421 02:17:11.073667 47728129008896 torch/_dynamo/guards.py:2611] [86/5] [__recompiles]     - L['___stack0'] == 923272.489632834                          
V0421 02:17:11.073667 47728129008896 torch/_dynamo/guards.py:2611] [86/5] [__recompiles]     - L['___stack0'] == 923270.662628611                          
V0421 02:17:11.073667 47728129008896 torch/_dynamo/guards.py:2611] [86/5] [__recompiles]     - L['___stack0'] == 923268.120714279                          
V0421 02:17:11.073667 47728129008896 torch/_dynamo/guards.py:2611] [86/5] [__recompiles]     - L['___stack0'] == 923268.024748108                          
V0421 02:17:11.073667 47728129008896 torch/_dynamo/guards.py:2611] [86/5] [__recompiles]     - L['___stack0'] == 923268.01162549                           
V0421 02:17:11.079853 47728129008896 torch/_dynamo/guards.py:2611] [86/6] [__recompiles] Recompiling function torch_dynamo_resume_in_hook_at_12 in /work2/09808/mp46753/frontera/ML_Project/VQGraph/benchmark_utils.py:12
V0421 02:17:11.079853 47728129008896 torch/_dynamo/guards.py:2611] [86/6] [__recompiles]     triggered by the following guard failure(s):
V0421 02:17:11.079853 47728129008896 torch/_dynamo/guards.py:2611] [86/6] [__recompiles]     - L['___stack0'] == 923272.558512393                          
V0421 02:17:11.079853 47728129008896 torch/_dynamo/guards.py:2611] [86/6] [__recompiles]     - L['___stack0'] == 923272.489632834                          
V0421 02:17:11.079853 47728129008896 torch/_dynamo/guards.py:2611] [86/6] [__recompiles]     - L['___stack0'] == 923270.662628611                          
V0421 02:17:11.079853 47728129008896 torch/_dynamo/guards.py:2611] [86/6] [__recompiles]     - L['___stack0'] == 923268.120714279                          
V0421 02:17:11.079853 47728129008896 torch/_dynamo/guards.py:2611] [86/6] [__recompiles]     - L['___stack0'] == 923268.024748108                          
V0421 02:17:11.079853 47728129008896 torch/_dynamo/guards.py:2611] [86/6] [__recompiles]     - L['___stack0'] == 923268.01162549                           
V0421 02:17:11.136657 47728129008896 torch/_dynamo/guards.py:2611] [57/1] [__recompiles] Recompiling function __setitem__ in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/view.py:82
V0421 02:17:11.136657 47728129008896 torch/_dynamo/guards.py:2611] [57/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:17:11.136657 47728129008896 torch/_dynamo/guards.py:2611] [57/1] [__recompiles]     - tensor 'L['val']' requires_grad mismatch. expected requires_grad=0
V0421 02:17:11.158624 47728129008896 torch/_dynamo/guards.py:2611] [61/1] [__recompiles] Recompiling function __setitem__ in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/frame.py:690
V0421 02:17:11.158624 47728129008896 torch/_dynamo/guards.py:2611] [61/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:17:11.158624 47728129008896 torch/_dynamo/guards.py:2611] [61/1] [__recompiles]     - tensor 'L['data']' requires_grad mismatch. expected requires_grad=0
V0421 02:17:11.177852 47728129008896 torch/_dynamo/guards.py:2611] [62/1] [__recompiles] Recompiling function update_column in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/frame.py:772
V0421 02:17:11.177852 47728129008896 torch/_dynamo/guards.py:2611] [62/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:17:11.177852 47728129008896 torch/_dynamo/guards.py:2611] [62/1] [__recompiles]     - tensor 'L['data']' requires_grad mismatch. expected requires_grad=0
V0421 02:17:11.197716 47728129008896 torch/_dynamo/guards.py:2611] [63/1] [__recompiles] Recompiling function create in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/frame.py:472
V0421 02:17:11.197716 47728129008896 torch/_dynamo/guards.py:2611] [63/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:17:11.197716 47728129008896 torch/_dynamo/guards.py:2611] [63/1] [__recompiles]     - tensor 'L['data']' requires_grad mismatch. expected requires_grad=0
V0421 02:17:11.215238 47728129008896 torch/_dynamo/guards.py:2611] [64/1] [__recompiles] Recompiling function __init__ in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/frame.py:210
V0421 02:17:11.215238 47728129008896 torch/_dynamo/guards.py:2611] [64/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:17:11.215238 47728129008896 torch/_dynamo/guards.py:2611] [64/1] [__recompiles]     - tensor 'L['storage']' requires_grad mismatch. expected requires_grad=0
V0421 02:17:11.234760 47728129008896 torch/_dynamo/guards.py:2611] [65/1] [__recompiles] Recompiling function infer_scheme in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/frame.py:151
V0421 02:17:11.234760 47728129008896 torch/_dynamo/guards.py:2611] [65/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:17:11.234760 47728129008896 torch/_dynamo/guards.py:2611] [65/1] [__recompiles]     - tensor 'L['tensor']' requires_grad mismatch. expected requires_grad=0
V0421 02:17:11.280163 47728129008896 torch/_dynamo/guards.py:2611] [68/1] [__recompiles] Recompiling function func in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/ops/spmm.py:213
V0421 02:17:11.280163 47728129008896 torch/_dynamo/guards.py:2611] [68/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:17:11.280163 47728129008896 torch/_dynamo/guards.py:2611] [68/1] [__recompiles]     - tensor 'L['x']' requires_grad mismatch. expected requires_grad=0
V0421 02:17:11.293690 47728129008896 torch/_dynamo/guards.py:2611] [70/1] [__recompiles] Recompiling function torch_dynamo_resume_in_gspmm_at_75 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/ops/spmm.py:75
V0421 02:17:11.293690 47728129008896 torch/_dynamo/guards.py:2611] [70/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:17:11.293690 47728129008896 torch/_dynamo/guards.py:2611] [70/1] [__recompiles]     - tensor 'L['lhs_data']' requires_grad mismatch. expected requires_grad=0
V0421 02:17:11.311445 47728129008896 torch/_dynamo/guards.py:2611] [74/1] [__recompiles] Recompiling function forward in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/sparse.py:163
V0421 02:17:11.311445 47728129008896 torch/_dynamo/guards.py:2611] [74/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:17:11.311445 47728129008896 torch/_dynamo/guards.py:2611] [74/1] [__recompiles]     - tensor 'L['X']' requires_grad mismatch. expected requires_grad=0
V0421 02:17:11.327587 47728129008896 torch/_dynamo/guards.py:2611] [77/1] [__recompiles] Recompiling function zeros in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:287
V0421 02:17:11.327587 47728129008896 torch/_dynamo/guards.py:2611] [77/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:17:11.327587 47728129008896 torch/_dynamo/guards.py:2611] [77/1] [__recompiles]     - L['shape'][1] == 500                                        
V0421 02:17:11.588969 47728129008896 torch/_dynamo/guards.py:2611] [6/4] [__recompiles] Recompiling function zerocopy_to_dgl_ndarray in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:432
V0421 02:17:11.588969 47728129008896 torch/_dynamo/guards.py:2611] [6/4] [__recompiles]     triggered by the following guard failure(s):
V0421 02:17:11.588969 47728129008896 torch/_dynamo/guards.py:2611] [6/4] [__recompiles]     - GLOBAL_STATE changed: grad_mode 
V0421 02:17:11.588969 47728129008896 torch/_dynamo/guards.py:2611] [6/4] [__recompiles]     - tensor 'L['data']' requires_grad mismatch. expected requires_grad=0
V0421 02:17:11.588969 47728129008896 torch/_dynamo/guards.py:2611] [6/4] [__recompiles]     - GLOBAL_STATE changed: grad_mode 
V0421 02:17:11.588969 47728129008896 torch/_dynamo/guards.py:2611] [6/4] [__recompiles]     - GLOBAL_STATE changed: grad_mode 
V0421 02:17:11.637009 47728129008896 torch/_dynamo/guards.py:2611] [79/1] [__recompiles] Recompiling function zerocopy_to_dgl_ndarray_for_write in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:454
V0421 02:17:11.637009 47728129008896 torch/_dynamo/guards.py:2611] [79/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:17:11.637009 47728129008896 torch/_dynamo/guards.py:2611] [79/1] [__recompiles]     - tensor 'L['input']' stride mismatch at index 0. expected 500, actual 128
V0421 02:17:11.682289 47728129008896 torch/_dynamo/guards.py:2611] [80/1] [__recompiles] Recompiling function check_is_view in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:441
V0421 02:17:11.682289 47728129008896 torch/_dynamo/guards.py:2611] [80/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:17:11.682289 47728129008896 torch/_dynamo/guards.py:2611] [80/1] [__recompiles]     - tensor 'L['input']' stride mismatch at index 0. expected 500, actual 128
V0421 02:17:11.693411 47728129008896 torch/_dynamo/guards.py:2611] [81/1] [__recompiles] Recompiling function torch_dynamo_resume_in_check_is_view_at_443 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:443
V0421 02:17:11.693411 47728129008896 torch/_dynamo/guards.py:2611] [81/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:17:11.693411 47728129008896 torch/_dynamo/guards.py:2611] [81/1] [__recompiles]     - tensor 'L['input']' stride mismatch at index 0. expected 500, actual 128
V0421 02:17:11.704936 47728129008896 torch/_dynamo/guards.py:2611] [82/1] [__recompiles] Recompiling function torch_dynamo_resume_in_zerocopy_to_dgl_ndarray_for_write_at_461 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:461
V0421 02:17:11.704936 47728129008896 torch/_dynamo/guards.py:2611] [82/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:17:11.704936 47728129008896 torch/_dynamo/guards.py:2611] [82/1] [__recompiles]     - tensor 'L['input']' stride mismatch at index 0. expected 500, actual 128
V0421 02:17:11.718982 47728129008896 torch/_dynamo/guards.py:2611] [83/1] [__recompiles] Recompiling function torch_dynamo_resume_in_forward_at_165 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/sparse.py:165
V0421 02:17:11.718982 47728129008896 torch/_dynamo/guards.py:2611] [83/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:17:11.718982 47728129008896 torch/_dynamo/guards.py:2611] [83/1] [__recompiles]     - tensor 'L['X']' requires_grad mismatch. expected requires_grad=0
V0421 02:17:11.758626 47728129008896 torch/_dynamo/guards.py:2611] [86/7] [__recompiles] Recompiling function torch_dynamo_resume_in_hook_at_12 in /work2/09808/mp46753/frontera/ML_Project/VQGraph/benchmark_utils.py:12
V0421 02:17:11.758626 47728129008896 torch/_dynamo/guards.py:2611] [86/7] [__recompiles]     triggered by the following guard failure(s):
V0421 02:17:11.758626 47728129008896 torch/_dynamo/guards.py:2611] [86/7] [__recompiles]     - L['___stack0'] == 923272.564695001                          
V0421 02:17:11.758626 47728129008896 torch/_dynamo/guards.py:2611] [86/7] [__recompiles]     - L['___stack0'] == 923272.558512393                          
V0421 02:17:11.758626 47728129008896 torch/_dynamo/guards.py:2611] [86/7] [__recompiles]     - L['___stack0'] == 923272.489632834                          
V0421 02:17:11.758626 47728129008896 torch/_dynamo/guards.py:2611] [86/7] [__recompiles]     - L['___stack0'] == 923270.662628611                          
V0421 02:17:11.758626 47728129008896 torch/_dynamo/guards.py:2611] [86/7] [__recompiles]     - L['___stack0'] == 923268.120714279                          
V0421 02:17:11.758626 47728129008896 torch/_dynamo/guards.py:2611] [86/7] [__recompiles]     - L['___stack0'] == 923268.024748108                          
V0421 02:17:11.758626 47728129008896 torch/_dynamo/guards.py:2611] [86/7] [__recompiles]     - L['___stack0'] == 923268.01162549                           
V0421 02:17:11.765062 47728129008896 torch/_dynamo/guards.py:2611] [86/8] [__recompiles] Recompiling function torch_dynamo_resume_in_hook_at_12 in /work2/09808/mp46753/frontera/ML_Project/VQGraph/benchmark_utils.py:12
V0421 02:17:11.765062 47728129008896 torch/_dynamo/guards.py:2611] [86/8] [__recompiles]     triggered by the following guard failure(s):
V0421 02:17:11.765062 47728129008896 torch/_dynamo/guards.py:2611] [86/8] [__recompiles]     - L['___stack0'] == 923273.243423015                          
V0421 02:17:11.765062 47728129008896 torch/_dynamo/guards.py:2611] [86/8] [__recompiles]     - L['___stack0'] == 923272.564695001                          
V0421 02:17:11.765062 47728129008896 torch/_dynamo/guards.py:2611] [86/8] [__recompiles]     - L['___stack0'] == 923272.558512393                          
V0421 02:17:11.765062 47728129008896 torch/_dynamo/guards.py:2611] [86/8] [__recompiles]     - L['___stack0'] == 923272.489632834                          
V0421 02:17:11.765062 47728129008896 torch/_dynamo/guards.py:2611] [86/8] [__recompiles]     - L['___stack0'] == 923270.662628611                          
V0421 02:17:11.765062 47728129008896 torch/_dynamo/guards.py:2611] [86/8] [__recompiles]     - L['___stack0'] == 923268.120714279                          
V0421 02:17:11.765062 47728129008896 torch/_dynamo/guards.py:2611] [86/8] [__recompiles]     - L['___stack0'] == 923268.024748108                          
V0421 02:17:11.765062 47728129008896 torch/_dynamo/guards.py:2611] [86/8] [__recompiles]     - L['___stack0'] == 923268.01162549                           
W0421 02:17:11.765200 47728129008896 torch/_dynamo/convert_frame.py:762] [86/8] torch._dynamo hit config.cache_size_limit (8)
W0421 02:17:11.765200 47728129008896 torch/_dynamo/convert_frame.py:762] [86/8]    function: 'torch_dynamo_resume_in_hook_at_12' (/work2/09808/mp46753/frontera/ML_Project/VQGraph/benchmark_utils.py:12)
W0421 02:17:11.765200 47728129008896 torch/_dynamo/convert_frame.py:762] [86/8]    last reason: L['___stack0'] == 923268.01162549                           
W0421 02:17:11.765200 47728129008896 torch/_dynamo/convert_frame.py:762] [86/8] To log all recompilation reasons, use TORCH_LOGS="recompiles".
W0421 02:17:11.765200 47728129008896 torch/_dynamo/convert_frame.py:762] [86/8] To diagnose recompilation issues, see https://pytorch.org/docs/main/torch.compiler_troubleshooting.html.
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/dataloading/dataloader.py:1144: DGLWarning: Dataloader CPU affinity opt is not enabled, consider switching it on (see enable_cpu_affinity() or CPU best practices for DGL [https://docs.dgl.ai/tutorials/cpu/cpu_best_practises.html])
  dgl_warning(
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:92: DGLWarning: Recommend creating graphs by `dgl.graph(data)` instead of `dgl.DGLGraph(data)`.
  dgl_warning(
V0421 02:17:13.012189 47728129008896 torch/_dynamo/guards.py:2611] [0/1] [__recompiles] Recompiling function forward in /work2/09808/mp46753/frontera/ML_Project/VQGraph/models.py:449
V0421 02:17:13.012189 47728129008896 torch/_dynamo/guards.py:2611] [0/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:17:13.012189 47728129008896 torch/_dynamo/guards.py:2611] [0/1] [__recompiles]     - tensor 'L['feats']' size mismatch at index 0. expected 684, actual 686
V0421 02:17:13.038642 47728129008896 torch/_dynamo/guards.py:2611] [1/1] [__recompiles] Recompiling function forward in /work2/09808/mp46753/frontera/ML_Project/VQGraph/models.py:154
V0421 02:17:13.038642 47728129008896 torch/_dynamo/guards.py:2611] [1/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:17:13.038642 47728129008896 torch/_dynamo/guards.py:2611] [1/1] [__recompiles]     - tensor 'L['feats']' size mismatch at index 0. expected 684, actual 686
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:92: DGLWarning: Recommend creating graphs by `dgl.graph(data)` instead of `dgl.DGLGraph(data)`.
  dgl_warning(
V0421 02:17:13.054327 47728129008896 torch/_dynamo/guards.py:2611] [9/1] [__recompiles] Recompiling function torch_dynamo_resume_in_forward_at_157 in /work2/09808/mp46753/frontera/ML_Project/VQGraph/models.py:157
V0421 02:17:13.054327 47728129008896 torch/_dynamo/guards.py:2611] [9/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:17:13.054327 47728129008896 torch/_dynamo/guards.py:2611] [9/1] [__recompiles]     - tensor 'L['h']' size mismatch at index 0. expected 684, actual 686
V0421 02:17:13.069743 47728129008896 torch/_dynamo/guards.py:2611] [13/1] [__recompiles] Recompiling function torch_dynamo_resume_in_forward_at_157 in /work2/09808/mp46753/frontera/ML_Project/VQGraph/models.py:157
V0421 02:17:13.069743 47728129008896 torch/_dynamo/guards.py:2611] [13/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:17:13.069743 47728129008896 torch/_dynamo/guards.py:2611] [13/1] [__recompiles]     - tensor 'L['h']' size mismatch at index 0. expected 684, actual 686
V0421 02:17:13.110375 47728129008896 torch/_dynamo/guards.py:2611] [4/3] [__recompiles] Recompiling function create_unitgraph_from_coo in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph_index.py:1254
V0421 02:17:13.110375 47728129008896 torch/_dynamo/guards.py:2611] [4/3] [__recompiles]     triggered by the following guard failure(s):
V0421 02:17:13.110375 47728129008896 torch/_dynamo/guards.py:2611] [4/3] [__recompiles]     - tensor 'L['row']' dispatch key set mismatch. expected DispatchKeySet(CPU, BackendSelect, ADInplaceOrView, AutogradCPU), actual DispatchKeySet(CUDA, BackendSelect, ADInplaceOrView, AutogradCUDA)
V0421 02:17:13.110375 47728129008896 torch/_dynamo/guards.py:2611] [4/3] [__recompiles]     - L['num_dst'] == 684                                         
V0421 02:17:13.110375 47728129008896 torch/_dynamo/guards.py:2611] [4/3] [__recompiles]     - L['num_dst'] == 684                                         
V0421 02:17:13.142002 47728129008896 torch/_dynamo/guards.py:2611] [8/3] [__recompiles] Recompiling function torch_dynamo_resume_in_create_unitgraph_from_coo_at_1296 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph_index.py:1296
V0421 02:17:13.142002 47728129008896 torch/_dynamo/guards.py:2611] [8/3] [__recompiles]     triggered by the following guard failure(s):
V0421 02:17:13.142002 47728129008896 torch/_dynamo/guards.py:2611] [8/3] [__recompiles]     - tensor 'L['col']' dispatch key set mismatch. expected DispatchKeySet(CPU, BackendSelect, ADInplaceOrView, AutogradCPU), actual DispatchKeySet(CUDA, BackendSelect, ADInplaceOrView, AutogradCUDA)
V0421 02:17:13.142002 47728129008896 torch/_dynamo/guards.py:2611] [8/3] [__recompiles]     - L['___stack2'] == 684                                       
V0421 02:17:13.142002 47728129008896 torch/_dynamo/guards.py:2611] [8/3] [__recompiles]     - L['___stack2'] == 684                                       
V0421 02:17:13.155781 47728129008896 torch/_dynamo/guards.py:2611] [15/1] [__recompiles] Recompiling function _dispatch in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/utils/internal.py:65
V0421 02:17:13.155781 47728129008896 torch/_dynamo/guards.py:2611] [15/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:17:13.155781 47728129008896 torch/_dynamo/guards.py:2611] [15/1] [__recompiles]     - L['data'][0] == 684                                         
V0421 02:17:16.381036 47728129008896 torch/_dynamo/guards.py:2611] [29/1] [__recompiles] Recompiling function torch_dynamo_resume_in_add_edges_at_572 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:572
V0421 02:17:16.381036 47728129008896 torch/_dynamo/guards.py:2611] [29/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:17:16.381036 47728129008896 torch/_dynamo/guards.py:2611] [29/1] [__recompiles]     - L['___stack0'] == 684                                       
V0421 02:17:16.398792 47728129008896 torch/_dynamo/guards.py:2611] [30/2] [__recompiles] Recompiling function torch_dynamo_resume_in_add_edges_at_573 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:573
V0421 02:17:16.398792 47728129008896 torch/_dynamo/guards.py:2611] [30/2] [__recompiles]     triggered by the following guard failure(s):
V0421 02:17:16.398792 47728129008896 torch/_dynamo/guards.py:2611] [30/2] [__recompiles]     - L['___stack0'] == 684                                       
V0421 02:17:16.398792 47728129008896 torch/_dynamo/guards.py:2611] [30/2] [__recompiles]     - tensor 'L['u']' size mismatch at index 0. expected 830, actual 839
V0421 02:17:16.645935 47728129008896 torch/_dynamo/guards.py:2611] [34/3] [__recompiles] Recompiling function cat in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:222
V0421 02:17:16.645935 47728129008896 torch/_dynamo/guards.py:2611] [34/3] [__recompiles]     triggered by the following guard failure(s):
V0421 02:17:16.645935 47728129008896 torch/_dynamo/guards.py:2611] [34/3] [__recompiles]     - 2 <= L['seq'][0].size()[0] <= 2147483645                      # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 02:17:16.645935 47728129008896 torch/_dynamo/guards.py:2611] [34/3] [__recompiles]     - tensor 'L['seq'][1]' size mismatch at index 0. expected 830, actual 839
V0421 02:17:16.645935 47728129008896 torch/_dynamo/guards.py:2611] [34/3] [__recompiles]     - tensor 'L['seq'][1]' size mismatch at index 0. expected 830, actual 839
V0421 02:17:16.984050 47728129008896 torch/_dynamo/guards.py:2611] [4/4] [__recompiles] Recompiling function create_unitgraph_from_coo in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph_index.py:1254
V0421 02:17:16.984050 47728129008896 torch/_dynamo/guards.py:2611] [4/4] [__recompiles]     triggered by the following guard failure(s):
V0421 02:17:16.984050 47728129008896 torch/_dynamo/guards.py:2611] [4/4] [__recompiles]     - tensor 'L['row']' size mismatch at index 0. expected 0, actual 839
V0421 02:17:16.984050 47728129008896 torch/_dynamo/guards.py:2611] [4/4] [__recompiles]     - tensor 'L['row']' dispatch key set mismatch. expected DispatchKeySet(CPU, BackendSelect, ADInplaceOrView, AutogradCPU), actual DispatchKeySet(CUDA, BackendSelect, ADInplaceOrView, AutogradCUDA)
V0421 02:17:16.984050 47728129008896 torch/_dynamo/guards.py:2611] [4/4] [__recompiles]     - L['num_dst'] == 684                                         
V0421 02:17:16.984050 47728129008896 torch/_dynamo/guards.py:2611] [4/4] [__recompiles]     - tensor 'L['row']' size mismatch at index 0. expected 0, actual 839
V0421 02:17:17.022626 47728129008896 torch/_dynamo/guards.py:2611] [8/4] [__recompiles] Recompiling function torch_dynamo_resume_in_create_unitgraph_from_coo_at_1296 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph_index.py:1296
V0421 02:17:17.022626 47728129008896 torch/_dynamo/guards.py:2611] [8/4] [__recompiles]     triggered by the following guard failure(s):
V0421 02:17:17.022626 47728129008896 torch/_dynamo/guards.py:2611] [8/4] [__recompiles]     - tensor 'L['col']' size mismatch at index 0. expected 0, actual 839
V0421 02:17:17.022626 47728129008896 torch/_dynamo/guards.py:2611] [8/4] [__recompiles]     - tensor 'L['col']' dispatch key set mismatch. expected DispatchKeySet(CPU, BackendSelect, ADInplaceOrView, AutogradCPU), actual DispatchKeySet(CUDA, BackendSelect, ADInplaceOrView, AutogradCUDA)
V0421 02:17:17.022626 47728129008896 torch/_dynamo/guards.py:2611] [8/4] [__recompiles]     - L['___stack2'] == 684                                       
V0421 02:17:17.022626 47728129008896 torch/_dynamo/guards.py:2611] [8/4] [__recompiles]     - tensor 'L['col']' size mismatch at index 0. expected 0, actual 839
V0421 02:17:17.043034 47728129008896 torch/_dynamo/guards.py:2611] [41/1] [__recompiles] Recompiling function spmatrix in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/sparse/sparse_matrix.py:763
V0421 02:17:17.043034 47728129008896 torch/_dynamo/guards.py:2611] [41/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:17:17.043034 47728129008896 torch/_dynamo/guards.py:2611] [41/1] [__recompiles]     - tensor 'L['indices']' stride mismatch at index 0. expected 2038, actual 2056
V0421 02:17:17.307661 47728129008896 torch/_dynamo/guards.py:2611] [47/1] [__recompiles] Recompiling function arange in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:406
V0421 02:17:17.307661 47728129008896 torch/_dynamo/guards.py:2611] [47/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:17:17.307661 47728129008896 torch/_dynamo/guards.py:2611] [47/1] [__recompiles]     - L['stop'] == 684                                            
V0421 02:17:17.528729 47728129008896 torch/_dynamo/guards.py:2611] [48/1] [__recompiles] Recompiling function in_degrees in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph_index.py:702
V0421 02:17:17.528729 47728129008896 torch/_dynamo/guards.py:2611] [48/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:17:17.528729 47728129008896 torch/_dynamo/guards.py:2611] [48/1] [__recompiles]     - tensor 'L['v']' size mismatch at index 0. expected 684, actual 686
V0421 02:17:17.546340 47728129008896 torch/_dynamo/guards.py:2611] [50/1] [__recompiles] Recompiling function has_nodes in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:2878
V0421 02:17:17.546340 47728129008896 torch/_dynamo/guards.py:2611] [50/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:17:17.546340 47728129008896 torch/_dynamo/guards.py:2611] [50/1] [__recompiles]     - tensor 'L['vid']' size mismatch at index 0. expected 684, actual 686
V0421 02:17:17.562312 47728129008896 torch/_dynamo/guards.py:2611] [51/1] [__recompiles] Recompiling function torch_dynamo_resume_in_has_nodes_at_2925 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:2925
V0421 02:17:17.562312 47728129008896 torch/_dynamo/guards.py:2611] [51/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:17:17.562312 47728129008896 torch/_dynamo/guards.py:2611] [51/1] [__recompiles]     - tensor 'L['___stack0']' size mismatch at index 0. expected 684, actual 686
V0421 02:17:17.805718 47728129008896 torch/_dynamo/guards.py:2611] [52/1] [__recompiles] Recompiling function torch_dynamo_resume_in_has_nodes_at_2926 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:2926
V0421 02:17:17.805718 47728129008896 torch/_dynamo/guards.py:2611] [52/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:17:17.805718 47728129008896 torch/_dynamo/guards.py:2611] [52/1] [__recompiles]     - tensor 'L['vid_tensor']' size mismatch at index 0. expected 684, actual 686
V0421 02:17:17.828830 47728129008896 torch/_dynamo/guards.py:2611] [53/1] [__recompiles] Recompiling function has_nodes in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph_index.py:444
V0421 02:17:17.828830 47728129008896 torch/_dynamo/guards.py:2611] [53/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:17:17.828830 47728129008896 torch/_dynamo/guards.py:2611] [53/1] [__recompiles]     - tensor 'L['vids']' size mismatch at index 0. expected 684, actual 686
V0421 02:17:17.845479 47728129008896 torch/_dynamo/guards.py:2611] [54/1] [__recompiles] Recompiling function torch_dynamo_resume_in_has_nodes_at_2930 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:2930
V0421 02:17:17.845479 47728129008896 torch/_dynamo/guards.py:2611] [54/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:17:17.845479 47728129008896 torch/_dynamo/guards.py:2611] [54/1] [__recompiles]     - tensor 'L['___stack0']' size mismatch at index 0. expected 684, actual 686
V0421 02:17:18.070065 47728129008896 torch/_dynamo/guards.py:2611] [55/1] [__recompiles] Recompiling function sum in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:152
V0421 02:17:18.070065 47728129008896 torch/_dynamo/guards.py:2611] [55/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:17:18.070065 47728129008896 torch/_dynamo/guards.py:2611] [55/1] [__recompiles]     - tensor 'L['input']' size mismatch at index 0. expected 684, actual 686
V0421 02:17:18.147855 47728129008896 torch/_dynamo/guards.py:2611] [57/2] [__recompiles] Recompiling function __setitem__ in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/view.py:82
V0421 02:17:18.147855 47728129008896 torch/_dynamo/guards.py:2611] [57/2] [__recompiles]     triggered by the following guard failure(s):
V0421 02:17:18.147855 47728129008896 torch/_dynamo/guards.py:2611] [57/2] [__recompiles]     - tensor 'L['val']' requires_grad mismatch. expected requires_grad=1
V0421 02:17:18.147855 47728129008896 torch/_dynamo/guards.py:2611] [57/2] [__recompiles]     - tensor 'L['val']' size mismatch at index 0. expected 684, actual 686
V0421 02:17:18.165615 47728129008896 torch/_dynamo/guards.py:2611] [61/2] [__recompiles] Recompiling function __setitem__ in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/frame.py:690
V0421 02:17:18.165615 47728129008896 torch/_dynamo/guards.py:2611] [61/2] [__recompiles]     triggered by the following guard failure(s):
V0421 02:17:18.165615 47728129008896 torch/_dynamo/guards.py:2611] [61/2] [__recompiles]     - tensor 'L['data']' requires_grad mismatch. expected requires_grad=1
V0421 02:17:18.165615 47728129008896 torch/_dynamo/guards.py:2611] [61/2] [__recompiles]     - tensor 'L['data']' size mismatch at index 0. expected 684, actual 686
V0421 02:17:18.183744 47728129008896 torch/_dynamo/guards.py:2611] [62/2] [__recompiles] Recompiling function update_column in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/frame.py:772
V0421 02:17:18.183744 47728129008896 torch/_dynamo/guards.py:2611] [62/2] [__recompiles]     triggered by the following guard failure(s):
V0421 02:17:18.183744 47728129008896 torch/_dynamo/guards.py:2611] [62/2] [__recompiles]     - tensor 'L['data']' requires_grad mismatch. expected requires_grad=1
V0421 02:17:18.183744 47728129008896 torch/_dynamo/guards.py:2611] [62/2] [__recompiles]     - tensor 'L['data']' size mismatch at index 0. expected 684, actual 686
V0421 02:17:18.202411 47728129008896 torch/_dynamo/guards.py:2611] [63/2] [__recompiles] Recompiling function create in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/frame.py:472
V0421 02:17:18.202411 47728129008896 torch/_dynamo/guards.py:2611] [63/2] [__recompiles]     triggered by the following guard failure(s):
V0421 02:17:18.202411 47728129008896 torch/_dynamo/guards.py:2611] [63/2] [__recompiles]     - tensor 'L['data']' requires_grad mismatch. expected requires_grad=1
V0421 02:17:18.202411 47728129008896 torch/_dynamo/guards.py:2611] [63/2] [__recompiles]     - tensor 'L['data']' size mismatch at index 0. expected 684, actual 686
V0421 02:17:18.218908 47728129008896 torch/_dynamo/guards.py:2611] [64/2] [__recompiles] Recompiling function __init__ in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/frame.py:210
V0421 02:17:18.218908 47728129008896 torch/_dynamo/guards.py:2611] [64/2] [__recompiles]     triggered by the following guard failure(s):
V0421 02:17:18.218908 47728129008896 torch/_dynamo/guards.py:2611] [64/2] [__recompiles]     - tensor 'L['storage']' requires_grad mismatch. expected requires_grad=1
V0421 02:17:18.218908 47728129008896 torch/_dynamo/guards.py:2611] [64/2] [__recompiles]     - tensor 'L['storage']' size mismatch at index 0. expected 684, actual 686
V0421 02:17:18.237483 47728129008896 torch/_dynamo/guards.py:2611] [65/2] [__recompiles] Recompiling function infer_scheme in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/frame.py:151
V0421 02:17:18.237483 47728129008896 torch/_dynamo/guards.py:2611] [65/2] [__recompiles]     triggered by the following guard failure(s):
V0421 02:17:18.237483 47728129008896 torch/_dynamo/guards.py:2611] [65/2] [__recompiles]     - tensor 'L['tensor']' requires_grad mismatch. expected requires_grad=1
V0421 02:17:18.237483 47728129008896 torch/_dynamo/guards.py:2611] [65/2] [__recompiles]     - tensor 'L['tensor']' size mismatch at index 0. expected 684, actual 686
V0421 02:17:18.280766 47728129008896 torch/_dynamo/guards.py:2611] [68/2] [__recompiles] Recompiling function func in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/ops/spmm.py:213
V0421 02:17:18.280766 47728129008896 torch/_dynamo/guards.py:2611] [68/2] [__recompiles]     triggered by the following guard failure(s):
V0421 02:17:18.280766 47728129008896 torch/_dynamo/guards.py:2611] [68/2] [__recompiles]     - tensor 'L['x']' requires_grad mismatch. expected requires_grad=1
V0421 02:17:18.280766 47728129008896 torch/_dynamo/guards.py:2611] [68/2] [__recompiles]     - tensor 'L['x']' size mismatch at index 0. expected 684, actual 686
V0421 02:17:18.294053 47728129008896 torch/_dynamo/guards.py:2611] [70/2] [__recompiles] Recompiling function torch_dynamo_resume_in_gspmm_at_75 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/ops/spmm.py:75
V0421 02:17:18.294053 47728129008896 torch/_dynamo/guards.py:2611] [70/2] [__recompiles]     triggered by the following guard failure(s):
V0421 02:17:18.294053 47728129008896 torch/_dynamo/guards.py:2611] [70/2] [__recompiles]     - tensor 'L['lhs_data']' requires_grad mismatch. expected requires_grad=1
V0421 02:17:18.294053 47728129008896 torch/_dynamo/guards.py:2611] [70/2] [__recompiles]     - tensor 'L['lhs_data']' size mismatch at index 0. expected 684, actual 686
V0421 02:17:18.311710 47728129008896 torch/_dynamo/guards.py:2611] [74/2] [__recompiles] Recompiling function forward in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/sparse.py:163
V0421 02:17:18.311710 47728129008896 torch/_dynamo/guards.py:2611] [74/2] [__recompiles]     triggered by the following guard failure(s):
V0421 02:17:18.311710 47728129008896 torch/_dynamo/guards.py:2611] [74/2] [__recompiles]     - tensor 'L['X']' requires_grad mismatch. expected requires_grad=1
V0421 02:17:18.311710 47728129008896 torch/_dynamo/guards.py:2611] [74/2] [__recompiles]     - tensor 'L['X']' size mismatch at index 0. expected 684, actual 686
V0421 02:17:18.327620 47728129008896 torch/_dynamo/guards.py:2611] [77/2] [__recompiles] Recompiling function zeros in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:287
V0421 02:17:18.327620 47728129008896 torch/_dynamo/guards.py:2611] [77/2] [__recompiles]     triggered by the following guard failure(s):
V0421 02:17:18.327620 47728129008896 torch/_dynamo/guards.py:2611] [77/2] [__recompiles]     - L['shape'][0] == 684                                        
V0421 02:17:18.327620 47728129008896 torch/_dynamo/guards.py:2611] [77/2] [__recompiles]     - L['shape'][0] == 684                                        
V0421 02:17:18.487960 47728129008896 torch/_dynamo/guards.py:2611] [79/2] [__recompiles] Recompiling function zerocopy_to_dgl_ndarray_for_write in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:454
V0421 02:17:18.487960 47728129008896 torch/_dynamo/guards.py:2611] [79/2] [__recompiles]     triggered by the following guard failure(s):
V0421 02:17:18.487960 47728129008896 torch/_dynamo/guards.py:2611] [79/2] [__recompiles]     - tensor 'L['input']' size mismatch at index 0. expected 684, actual 686
V0421 02:17:18.487960 47728129008896 torch/_dynamo/guards.py:2611] [79/2] [__recompiles]     - tensor 'L['input']' size mismatch at index 0. expected 684, actual 686
V0421 02:17:18.528023 47728129008896 torch/_dynamo/guards.py:2611] [80/2] [__recompiles] Recompiling function check_is_view in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:441
V0421 02:17:18.528023 47728129008896 torch/_dynamo/guards.py:2611] [80/2] [__recompiles]     triggered by the following guard failure(s):
V0421 02:17:18.528023 47728129008896 torch/_dynamo/guards.py:2611] [80/2] [__recompiles]     - tensor 'L['input']' size mismatch at index 0. expected 684, actual 686
V0421 02:17:18.528023 47728129008896 torch/_dynamo/guards.py:2611] [80/2] [__recompiles]     - tensor 'L['input']' size mismatch at index 0. expected 684, actual 686
V0421 02:17:18.539406 47728129008896 torch/_dynamo/guards.py:2611] [81/2] [__recompiles] Recompiling function torch_dynamo_resume_in_check_is_view_at_443 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:443
V0421 02:17:18.539406 47728129008896 torch/_dynamo/guards.py:2611] [81/2] [__recompiles]     triggered by the following guard failure(s):
V0421 02:17:18.539406 47728129008896 torch/_dynamo/guards.py:2611] [81/2] [__recompiles]     - tensor 'L['input']' size mismatch at index 0. expected 684, actual 686
V0421 02:17:18.539406 47728129008896 torch/_dynamo/guards.py:2611] [81/2] [__recompiles]     - tensor 'L['input']' size mismatch at index 0. expected 684, actual 686
V0421 02:17:18.552049 47728129008896 torch/_dynamo/guards.py:2611] [82/2] [__recompiles] Recompiling function torch_dynamo_resume_in_zerocopy_to_dgl_ndarray_for_write_at_461 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:461
V0421 02:17:18.552049 47728129008896 torch/_dynamo/guards.py:2611] [82/2] [__recompiles]     triggered by the following guard failure(s):
V0421 02:17:18.552049 47728129008896 torch/_dynamo/guards.py:2611] [82/2] [__recompiles]     - tensor 'L['input']' size mismatch at index 0. expected 684, actual 686
V0421 02:17:18.552049 47728129008896 torch/_dynamo/guards.py:2611] [82/2] [__recompiles]     - tensor 'L['input']' size mismatch at index 0. expected 684, actual 686
V0421 02:17:18.568384 47728129008896 torch/_dynamo/guards.py:2611] [88/1] [__recompiles] Recompiling function forward in /work2/09808/mp46753/frontera/ML_Project/VQGraph/vq.py:569
V0421 02:17:18.568384 47728129008896 torch/_dynamo/guards.py:2611] [88/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:17:18.568384 47728129008896 torch/_dynamo/guards.py:2611] [88/1] [__recompiles]     - tensor 'L['x']' size mismatch at index 0. expected 684, actual 686
V0421 02:17:18.657648 47728129008896 torch/_dynamo/guards.py:2611] [89/1] [__recompiles] Recompiling function torch_dynamo_resume_in_forward_at_590 in /work2/09808/mp46753/frontera/ML_Project/VQGraph/vq.py:590
V0421 02:17:18.657648 47728129008896 torch/_dynamo/guards.py:2611] [89/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:17:18.657648 47728129008896 torch/_dynamo/guards.py:2611] [89/1] [__recompiles]     - tensor 'L['___stack0']' size mismatch at index 0. expected 684, actual 686
V0421 02:17:18.718366 47728129008896 torch/_dynamo/guards.py:2611] [90/1] [__recompiles] Recompiling function forward in /work2/09808/mp46753/frontera/ML_Project/VQGraph/vq.py:418
V0421 02:17:18.718366 47728129008896 torch/_dynamo/guards.py:2611] [90/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:17:18.718366 47728129008896 torch/_dynamo/guards.py:2611] [90/1] [__recompiles]     - tensor 'L['x']' size mismatch at index 0. expected 684, actual 686
V0421 02:17:19.669842 47728129008896 torch/_dynamo/guards.py:2611] [93/1] [__recompiles] Recompiling function torch_dynamo_resume_in_forward_at_432 in /work2/09808/mp46753/frontera/ML_Project/VQGraph/vq.py:432
V0421 02:17:19.669842 47728129008896 torch/_dynamo/guards.py:2611] [93/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:17:19.669842 47728129008896 torch/_dynamo/guards.py:2611] [93/1] [__recompiles]     - L['shape'][1] == 684                                        
V0421 02:17:21.527832 47728129008896 torch/_dynamo/guards.py:2611] [94/1] [__recompiles] Recompiling function torch_dynamo_resume_in_forward_at_595 in /work2/09808/mp46753/frontera/ML_Project/VQGraph/vq.py:595
V0421 02:17:21.527832 47728129008896 torch/_dynamo/guards.py:2611] [94/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:17:21.527832 47728129008896 torch/_dynamo/guards.py:2611] [94/1] [__recompiles]     - tensor 'L['x']' size mismatch at index 0. expected 684, actual 686
V0421 02:17:21.869273 47728129008896 torch/_dynamo/guards.py:2611] [95/1] [__recompiles] Recompiling function torch_dynamo_resume_in_forward_at_604 in /work2/09808/mp46753/frontera/ML_Project/VQGraph/vq.py:604
V0421 02:17:21.869273 47728129008896 torch/_dynamo/guards.py:2611] [95/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:17:21.869273 47728129008896 torch/_dynamo/guards.py:2611] [95/1] [__recompiles]     - tensor 'L['x']' size mismatch at index 0. expected 684, actual 686
V0421 02:17:22.803454 47728129008896 torch/_dynamo/guards.py:2611] [96/1] [__recompiles] Recompiling function torch_dynamo_resume_in_forward_at_648 in /work2/09808/mp46753/frontera/ML_Project/VQGraph/vq.py:648
V0421 02:17:22.803454 47728129008896 torch/_dynamo/guards.py:2611] [96/1] [__recompiles]     triggered by the following guard failure(s):
V0421 02:17:22.803454 47728129008896 torch/_dynamo/guards.py:2611] [96/1] [__recompiles]     - tensor 'L['___stack0']' size mismatch at index 0. expected 684, actual 686
V0421 02:17:22.903810 47728129008896 torch/_dynamo/guards.py:2611] [57/3] [__recompiles] Recompiling function __setitem__ in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/view.py:82
V0421 02:17:22.903810 47728129008896 torch/_dynamo/guards.py:2611] [57/3] [__recompiles]     triggered by the following guard failure(s):
V0421 02:17:22.903810 47728129008896 torch/_dynamo/guards.py:2611] [57/3] [__recompiles]     - tensor 'L['val']' requires_grad mismatch. expected requires_grad=0
V0421 02:17:22.903810 47728129008896 torch/_dynamo/guards.py:2611] [57/3] [__recompiles]     - tensor 'L['val']' size mismatch at index 0. expected 684, actual 686
V0421 02:17:22.903810 47728129008896 torch/_dynamo/guards.py:2611] [57/3] [__recompiles]     - tensor 'L['val']' requires_grad mismatch. expected requires_grad=0
V0421 02:17:22.924612 47728129008896 torch/_dynamo/guards.py:2611] [61/3] [__recompiles] Recompiling function __setitem__ in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/frame.py:690
V0421 02:17:22.924612 47728129008896 torch/_dynamo/guards.py:2611] [61/3] [__recompiles]     triggered by the following guard failure(s):
V0421 02:17:22.924612 47728129008896 torch/_dynamo/guards.py:2611] [61/3] [__recompiles]     - tensor 'L['data']' requires_grad mismatch. expected requires_grad=0
V0421 02:17:22.924612 47728129008896 torch/_dynamo/guards.py:2611] [61/3] [__recompiles]     - tensor 'L['data']' size mismatch at index 0. expected 684, actual 686
V0421 02:17:22.924612 47728129008896 torch/_dynamo/guards.py:2611] [61/3] [__recompiles]     - tensor 'L['data']' requires_grad mismatch. expected requires_grad=0
V0421 02:17:22.945186 47728129008896 torch/_dynamo/guards.py:2611] [62/3] [__recompiles] Recompiling function update_column in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/frame.py:772
V0421 02:17:22.945186 47728129008896 torch/_dynamo/guards.py:2611] [62/3] [__recompiles]     triggered by the following guard failure(s):
V0421 02:17:22.945186 47728129008896 torch/_dynamo/guards.py:2611] [62/3] [__recompiles]     - tensor 'L['data']' requires_grad mismatch. expected requires_grad=0
V0421 02:17:22.945186 47728129008896 torch/_dynamo/guards.py:2611] [62/3] [__recompiles]     - tensor 'L['data']' size mismatch at index 0. expected 684, actual 686
V0421 02:17:22.945186 47728129008896 torch/_dynamo/guards.py:2611] [62/3] [__recompiles]     - tensor 'L['data']' requires_grad mismatch. expected requires_grad=0
V0421 02:17:22.966100 47728129008896 torch/_dynamo/guards.py:2611] [63/3] [__recompiles] Recompiling function create in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/frame.py:472
V0421 02:17:22.966100 47728129008896 torch/_dynamo/guards.py:2611] [63/3] [__recompiles]     triggered by the following guard failure(s):
V0421 02:17:22.966100 47728129008896 torch/_dynamo/guards.py:2611] [63/3] [__recompiles]     - tensor 'L['data']' requires_grad mismatch. expected requires_grad=0
V0421 02:17:22.966100 47728129008896 torch/_dynamo/guards.py:2611] [63/3] [__recompiles]     - tensor 'L['data']' size mismatch at index 0. expected 684, actual 686
V0421 02:17:22.966100 47728129008896 torch/_dynamo/guards.py:2611] [63/3] [__recompiles]     - tensor 'L['data']' requires_grad mismatch. expected requires_grad=0
V0421 02:17:22.984984 47728129008896 torch/_dynamo/guards.py:2611] [64/3] [__recompiles] Recompiling function __init__ in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/frame.py:210
V0421 02:17:22.984984 47728129008896 torch/_dynamo/guards.py:2611] [64/3] [__recompiles]     triggered by the following guard failure(s):
V0421 02:17:22.984984 47728129008896 torch/_dynamo/guards.py:2611] [64/3] [__recompiles]     - tensor 'L['storage']' requires_grad mismatch. expected requires_grad=0
V0421 02:17:22.984984 47728129008896 torch/_dynamo/guards.py:2611] [64/3] [__recompiles]     - tensor 'L['storage']' size mismatch at index 0. expected 684, actual 686
V0421 02:17:22.984984 47728129008896 torch/_dynamo/guards.py:2611] [64/3] [__recompiles]     - tensor 'L['storage']' requires_grad mismatch. expected requires_grad=0
V0421 02:17:23.005849 47728129008896 torch/_dynamo/guards.py:2611] [65/3] [__recompiles] Recompiling function infer_scheme in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/frame.py:151
V0421 02:17:23.005849 47728129008896 torch/_dynamo/guards.py:2611] [65/3] [__recompiles]     triggered by the following guard failure(s):
V0421 02:17:23.005849 47728129008896 torch/_dynamo/guards.py:2611] [65/3] [__recompiles]     - tensor 'L['tensor']' requires_grad mismatch. expected requires_grad=0
V0421 02:17:23.005849 47728129008896 torch/_dynamo/guards.py:2611] [65/3] [__recompiles]     - tensor 'L['tensor']' size mismatch at index 0. expected 684, actual 686
V0421 02:17:23.005849 47728129008896 torch/_dynamo/guards.py:2611] [65/3] [__recompiles]     - tensor 'L['tensor']' requires_grad mismatch. expected requires_grad=0
V0421 02:17:23.053944 47728129008896 torch/_dynamo/guards.py:2611] [68/3] [__recompiles] Recompiling function func in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/ops/spmm.py:213
V0421 02:17:23.053944 47728129008896 torch/_dynamo/guards.py:2611] [68/3] [__recompiles]     triggered by the following guard failure(s):
V0421 02:17:23.053944 47728129008896 torch/_dynamo/guards.py:2611] [68/3] [__recompiles]     - tensor 'L['x']' requires_grad mismatch. expected requires_grad=0
V0421 02:17:23.053944 47728129008896 torch/_dynamo/guards.py:2611] [68/3] [__recompiles]     - tensor 'L['x']' size mismatch at index 0. expected 684, actual 686
V0421 02:17:23.053944 47728129008896 torch/_dynamo/guards.py:2611] [68/3] [__recompiles]     - tensor 'L['x']' requires_grad mismatch. expected requires_grad=0
V0421 02:17:23.068422 47728129008896 torch/_dynamo/guards.py:2611] [70/3] [__recompiles] Recompiling function torch_dynamo_resume_in_gspmm_at_75 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/ops/spmm.py:75
V0421 02:17:23.068422 47728129008896 torch/_dynamo/guards.py:2611] [70/3] [__recompiles]     triggered by the following guard failure(s):
V0421 02:17:23.068422 47728129008896 torch/_dynamo/guards.py:2611] [70/3] [__recompiles]     - tensor 'L['lhs_data']' requires_grad mismatch. expected requires_grad=0
V0421 02:17:23.068422 47728129008896 torch/_dynamo/guards.py:2611] [70/3] [__recompiles]     - tensor 'L['lhs_data']' size mismatch at index 0. expected 684, actual 686
V0421 02:17:23.068422 47728129008896 torch/_dynamo/guards.py:2611] [70/3] [__recompiles]     - tensor 'L['lhs_data']' requires_grad mismatch. expected requires_grad=0
V0421 02:17:23.087150 47728129008896 torch/_dynamo/guards.py:2611] [74/3] [__recompiles] Recompiling function forward in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/sparse.py:163
V0421 02:17:23.087150 47728129008896 torch/_dynamo/guards.py:2611] [74/3] [__recompiles]     triggered by the following guard failure(s):
V0421 02:17:23.087150 47728129008896 torch/_dynamo/guards.py:2611] [74/3] [__recompiles]     - tensor 'L['X']' requires_grad mismatch. expected requires_grad=0
V0421 02:17:23.087150 47728129008896 torch/_dynamo/guards.py:2611] [74/3] [__recompiles]     - tensor 'L['X']' size mismatch at index 0. expected 684, actual 686
V0421 02:17:23.087150 47728129008896 torch/_dynamo/guards.py:2611] [74/3] [__recompiles]     - tensor 'L['X']' requires_grad mismatch. expected requires_grad=0
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/dataloading/dataloader.py:1144: DGLWarning: Dataloader CPU affinity opt is not enabled, consider switching it on (see enable_cpu_affinity() or CPU best practices for DGL [https://docs.dgl.ai/tutorials/cpu/cpu_best_practises.html])
  dgl_warning(
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:92: DGLWarning: Recommend creating graphs by `dgl.graph(data)` instead of `dgl.DGLGraph(data)`.
  dgl_warning(
V0421 02:17:23.634530 47728129008896 torch/_dynamo/guards.py:2611] [15/2] [__recompiles] Recompiling function _dispatch in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/utils/internal.py:65
V0421 02:17:23.634530 47728129008896 torch/_dynamo/guards.py:2611] [15/2] [__recompiles]     triggered by the following guard failure(s):
V0421 02:17:23.634530 47728129008896 torch/_dynamo/guards.py:2611] [15/2] [__recompiles]     - L['data'][0] == 686                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 02:17:23.634530 47728129008896 torch/_dynamo/guards.py:2611] [15/2] [__recompiles]     - L['data'][0] == 684                                         
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/dataloading/dataloader.py:1144: DGLWarning: Dataloader CPU affinity opt is not enabled, consider switching it on (see enable_cpu_affinity() or CPU best practices for DGL [https://docs.dgl.ai/tutorials/cpu/cpu_best_practises.html])
  dgl_warning(
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:92: DGLWarning: Recommend creating graphs by `dgl.graph(data)` instead of `dgl.DGLGraph(data)`.
  dgl_warning(
V0421 02:17:27.528917 47728129008896 torch/_dynamo/guards.py:2611] [15/3] [__recompiles] Recompiling function _dispatch in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/utils/internal.py:65
V0421 02:17:27.528917 47728129008896 torch/_dynamo/guards.py:2611] [15/3] [__recompiles]     triggered by the following guard failure(s):
V0421 02:17:27.528917 47728129008896 torch/_dynamo/guards.py:2611] [15/3] [__recompiles]     - L['data'][0] == 687                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 02:17:27.528917 47728129008896 torch/_dynamo/guards.py:2611] [15/3] [__recompiles]     - L['data'][0] == 686                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 02:17:27.528917 47728129008896 torch/_dynamo/guards.py:2611] [15/3] [__recompiles]     - L['data'][0] == 684                                         
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/dataloading/dataloader.py:1144: DGLWarning: Dataloader CPU affinity opt is not enabled, consider switching it on (see enable_cpu_affinity() or CPU best practices for DGL [https://docs.dgl.ai/tutorials/cpu/cpu_best_practises.html])
  dgl_warning(
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:92: DGLWarning: Recommend creating graphs by `dgl.graph(data)` instead of `dgl.DGLGraph(data)`.
  dgl_warning(
V0421 02:17:31.681684 47728129008896 torch/_dynamo/guards.py:2611] [15/4] [__recompiles] Recompiling function _dispatch in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/utils/internal.py:65
V0421 02:17:31.681684 47728129008896 torch/_dynamo/guards.py:2611] [15/4] [__recompiles]     triggered by the following guard failure(s):
V0421 02:17:31.681684 47728129008896 torch/_dynamo/guards.py:2611] [15/4] [__recompiles]     - L['data'][0] == 687                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 02:17:31.681684 47728129008896 torch/_dynamo/guards.py:2611] [15/4] [__recompiles]     - L['data'][0] == 704                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 02:17:31.681684 47728129008896 torch/_dynamo/guards.py:2611] [15/4] [__recompiles]     - L['data'][0] == 686                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 02:17:31.681684 47728129008896 torch/_dynamo/guards.py:2611] [15/4] [__recompiles]     - L['data'][0] == 684                                         
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/dataloading/dataloader.py:1144: DGLWarning: Dataloader CPU affinity opt is not enabled, consider switching it on (see enable_cpu_affinity() or CPU best practices for DGL [https://docs.dgl.ai/tutorials/cpu/cpu_best_practises.html])
  dgl_warning(
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:92: DGLWarning: Recommend creating graphs by `dgl.graph(data)` instead of `dgl.DGLGraph(data)`.
  dgl_warning(
V0421 02:17:35.378535 47728129008896 torch/_dynamo/guards.py:2611] [15/5] [__recompiles] Recompiling function _dispatch in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/utils/internal.py:65
V0421 02:17:35.378535 47728129008896 torch/_dynamo/guards.py:2611] [15/5] [__recompiles]     triggered by the following guard failure(s):
V0421 02:17:35.378535 47728129008896 torch/_dynamo/guards.py:2611] [15/5] [__recompiles]     - L['data'][0] == 692                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 02:17:35.378535 47728129008896 torch/_dynamo/guards.py:2611] [15/5] [__recompiles]     - L['data'][0] == 687                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 02:17:35.378535 47728129008896 torch/_dynamo/guards.py:2611] [15/5] [__recompiles]     - L['data'][0] == 704                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 02:17:35.378535 47728129008896 torch/_dynamo/guards.py:2611] [15/5] [__recompiles]     - L['data'][0] == 686                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 02:17:35.378535 47728129008896 torch/_dynamo/guards.py:2611] [15/5] [__recompiles]     - L['data'][0] == 684                                         
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/dataloading/dataloader.py:1144: DGLWarning: Dataloader CPU affinity opt is not enabled, consider switching it on (see enable_cpu_affinity() or CPU best practices for DGL [https://docs.dgl.ai/tutorials/cpu/cpu_best_practises.html])
  dgl_warning(
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:92: DGLWarning: Recommend creating graphs by `dgl.graph(data)` instead of `dgl.DGLGraph(data)`.
  dgl_warning(
V0421 02:17:39.085136 47728129008896 torch/_dynamo/guards.py:2611] [15/6] [__recompiles] Recompiling function _dispatch in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/utils/internal.py:65
V0421 02:17:39.085136 47728129008896 torch/_dynamo/guards.py:2611] [15/6] [__recompiles]     triggered by the following guard failure(s):
V0421 02:17:39.085136 47728129008896 torch/_dynamo/guards.py:2611] [15/6] [__recompiles]     - L['data'][0] == 682                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 02:17:39.085136 47728129008896 torch/_dynamo/guards.py:2611] [15/6] [__recompiles]     - L['data'][0] == 692                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 02:17:39.085136 47728129008896 torch/_dynamo/guards.py:2611] [15/6] [__recompiles]     - L['data'][0] == 687                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 02:17:39.085136 47728129008896 torch/_dynamo/guards.py:2611] [15/6] [__recompiles]     - L['data'][0] == 704                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 02:17:39.085136 47728129008896 torch/_dynamo/guards.py:2611] [15/6] [__recompiles]     - L['data'][0] == 686                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 02:17:39.085136 47728129008896 torch/_dynamo/guards.py:2611] [15/6] [__recompiles]     - L['data'][0] == 684                                         
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/dataloading/dataloader.py:1144: DGLWarning: Dataloader CPU affinity opt is not enabled, consider switching it on (see enable_cpu_affinity() or CPU best practices for DGL [https://docs.dgl.ai/tutorials/cpu/cpu_best_practises.html])
  dgl_warning(
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:92: DGLWarning: Recommend creating graphs by `dgl.graph(data)` instead of `dgl.DGLGraph(data)`.
  dgl_warning(
V0421 02:17:42.740125 47728129008896 torch/_dynamo/guards.py:2611] [15/7] [__recompiles] Recompiling function _dispatch in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/utils/internal.py:65
V0421 02:17:42.740125 47728129008896 torch/_dynamo/guards.py:2611] [15/7] [__recompiles]     triggered by the following guard failure(s):
V0421 02:17:42.740125 47728129008896 torch/_dynamo/guards.py:2611] [15/7] [__recompiles]     - L['data'][0] == 679                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 02:17:42.740125 47728129008896 torch/_dynamo/guards.py:2611] [15/7] [__recompiles]     - L['data'][0] == 682                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 02:17:42.740125 47728129008896 torch/_dynamo/guards.py:2611] [15/7] [__recompiles]     - L['data'][0] == 692                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 02:17:42.740125 47728129008896 torch/_dynamo/guards.py:2611] [15/7] [__recompiles]     - L['data'][0] == 687                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 02:17:42.740125 47728129008896 torch/_dynamo/guards.py:2611] [15/7] [__recompiles]     - L['data'][0] == 704                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 02:17:42.740125 47728129008896 torch/_dynamo/guards.py:2611] [15/7] [__recompiles]     - L['data'][0] == 686                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 02:17:42.740125 47728129008896 torch/_dynamo/guards.py:2611] [15/7] [__recompiles]     - L['data'][0] == 684                                         
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/dataloading/dataloader.py:1144: DGLWarning: Dataloader CPU affinity opt is not enabled, consider switching it on (see enable_cpu_affinity() or CPU best practices for DGL [https://docs.dgl.ai/tutorials/cpu/cpu_best_practises.html])
  dgl_warning(
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:92: DGLWarning: Recommend creating graphs by `dgl.graph(data)` instead of `dgl.DGLGraph(data)`.
  dgl_warning(
V0421 02:17:46.426398 47728129008896 torch/_dynamo/guards.py:2611] [15/8] [__recompiles] Recompiling function _dispatch in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/utils/internal.py:65
V0421 02:17:46.426398 47728129008896 torch/_dynamo/guards.py:2611] [15/8] [__recompiles]     triggered by the following guard failure(s):
V0421 02:17:46.426398 47728129008896 torch/_dynamo/guards.py:2611] [15/8] [__recompiles]     - L['data'][0] == 685                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 02:17:46.426398 47728129008896 torch/_dynamo/guards.py:2611] [15/8] [__recompiles]     - L['data'][0] == 679                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 02:17:46.426398 47728129008896 torch/_dynamo/guards.py:2611] [15/8] [__recompiles]     - L['data'][0] == 682                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 02:17:46.426398 47728129008896 torch/_dynamo/guards.py:2611] [15/8] [__recompiles]     - L['data'][0] == 692                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 02:17:46.426398 47728129008896 torch/_dynamo/guards.py:2611] [15/8] [__recompiles]     - L['data'][0] == 687                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 02:17:46.426398 47728129008896 torch/_dynamo/guards.py:2611] [15/8] [__recompiles]     - L['data'][0] == 704                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 02:17:46.426398 47728129008896 torch/_dynamo/guards.py:2611] [15/8] [__recompiles]     - L['data'][0] == 686                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 02:17:46.426398 47728129008896 torch/_dynamo/guards.py:2611] [15/8] [__recompiles]     - L['data'][0] == 684                                         
W0421 02:17:46.426587 47728129008896 torch/_dynamo/convert_frame.py:762] [15/8] torch._dynamo hit config.cache_size_limit (8)
W0421 02:17:46.426587 47728129008896 torch/_dynamo/convert_frame.py:762] [15/8]    function: '_dispatch' (/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/utils/internal.py:65)
W0421 02:17:46.426587 47728129008896 torch/_dynamo/convert_frame.py:762] [15/8]    last reason: L['data'][0] == 684                                         
W0421 02:17:46.426587 47728129008896 torch/_dynamo/convert_frame.py:762] [15/8] To log all recompilation reasons, use TORCH_LOGS="recompiles".
W0421 02:17:46.426587 47728129008896 torch/_dynamo/convert_frame.py:762] [15/8] To diagnose recompilation issues, see https://pytorch.org/docs/main/torch.compiler_troubleshooting.html.
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/dataloading/dataloader.py:1144: DGLWarning: Dataloader CPU affinity opt is not enabled, consider switching it on (see enable_cpu_affinity() or CPU best practices for DGL [https://docs.dgl.ai/tutorials/cpu/cpu_best_practises.html])
  dgl_warning(
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:92: DGLWarning: Recommend creating graphs by `dgl.graph(data)` instead of `dgl.DGLGraph(data)`.
  dgl_warning(
Compiling model with torch.compile()...
Saved sparsity snapshot to outputs_compiled/tran/pubmed/SAGE/seed_0/sparsity_before.csv
                               layer_name  num_elements  sparsity
0  _orig_mod.encoder.graph_layer_1.weight        250000       0.0
1    _orig_mod.encoder.graph_layer_1.bias           500     100.0
2  _orig_mod.encoder.graph_layer_2.weight         64000       0.0
3    _orig_mod.encoder.graph_layer_2.bias           128     100.0
4      _orig_mod.encoder.decoder_1.weight        250000       0.0
5        _orig_mod.encoder.decoder_1.bias           500       0.0
6      _orig_mod.encoder.decoder_2.weight        250000       0.0
7        _orig_mod.encoder.decoder_2.bias           500       0.0
8         _orig_mod.encoder.linear.weight           384       0.0
9           _orig_mod.encoder.linear.bias             3       0.0
Saved sparsity snapshot to outputs_compiled/tran/pubmed/SAGE/seed_0/sparsity_after.csv
                               layer_name  num_elements  sparsity
0  _orig_mod.encoder.graph_layer_1.weight        250000    0.0000
1    _orig_mod.encoder.graph_layer_1.bias           500    0.0000
2  _orig_mod.encoder.graph_layer_2.weight         64000    0.0000
3    _orig_mod.encoder.graph_layer_2.bias           128    0.0000
4      _orig_mod.encoder.decoder_1.weight        250000    0.0000
5        _orig_mod.encoder.decoder_1.bias           500    0.0000
6      _orig_mod.encoder.decoder_2.weight        250000    0.0004
7        _orig_mod.encoder.decoder_2.bias           500    0.0000
8         _orig_mod.encoder.linear.weight           384    0.0000
9           _orig_mod.encoder.linear.bias             3    0.0000
Saved forward timing report to outputs_compiled/tran/pubmed/SAGE/seed_0/forward_times.csv
                         layer_name  avg_forward_time_ms  num_calls
0         _orig_mod.encoder.dropout             0.000488      20039
1   _orig_mod.encoder.graph_layer_1             0.000711      20039
2   _orig_mod.encoder.graph_layer_2             0.000616      20039
3       _orig_mod.encoder.decoder_1             0.000531      20039
4       _orig_mod.encoder.decoder_2             0.000519      20039
5          _orig_mod.encoder.linear             0.000280      20039
6   _orig_mod.encoder.vq.project_in             0.000481      20039
7  _orig_mod.encoder.vq.project_out             0.000585      20039
8    _orig_mod.encoder.vq._codebook             0.000613      20039
CPU Memory Usage: 1272.87 MB
Max GPU Memory Used: 1244.65 MB
1244.59 MB
